tkg/client/management_components.go

```diff
@@ -72,8 +73,18 @@ func (c *TkgClient) InstallOrUpgradeManagementComponents(kubeconfig, kubecontext
 
 	managementPackageVersion = strings.TrimLeft(managementPackageVersion, "v")
 
+	var addonsManagerPackageVersion string
+	if upgrade {
+		addonsManagerPackageVersion, err = c.GetAddonsManagerPackageversion(managementPackageVersion)
+		if err != nil {
+			return err
+		}
+	} else {
+		addonsManagerPackageVersion = managementPackageVersion
```

Comment:
I think setting addonsManagerPackageVersion to env var _ADDONS_MANAGER_PACKAGE_VERSION would make it consistent.

i.e. 
```
else {
            addonsManagerPackageVersion = managementPackageVersion
            envDefinedVersion := os.Getenv("_ADDONS_MANAGER_PACKAGE_VERSION")
	   if envDefinedVersion != "" {
		addonsManagerPackageVersion = envDefinedVersion
	   }
}
```

Reply:
done